setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+####################################################################
+# ### setup.py                                                   ###
+####################################################################
+# ### Author: SAS Institute Inc.                                 ###
+####################################################################
+#                                                                ###
+# Copyright (c) 2021, SAS Institute Inc., Cary, NC, USA.         ###
+# All Rights Reserved.                                           ###
+# SPDX-License-Identifier: Apache-2.0                            ###
+#                                                                ###
+####################################################################
+
+from setuptools import find_packages, setup
+
+# get the install requirements from requirements.txt
+with open("requirements.txt") as requirements_file:
+    install_requirements = requirements_file.read().splitlines()
+
+setup(
+    # UPDATE VERSION WITH EACH RELEASE
+    ##################################
+    version="1.3.1",
+    ##################################
+    name="viya4-ark",
+    author="SAS Institute Inc.",
+    author_email="Josh.Woods@sas.com",
```

Comment:
your email shouldn't go here unless you like spam

Reply:
Sounds good. Will change.